<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWindow</class>
 <widget class="QDialog" name="RegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>20</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:transparent;
border-style: solid;
color: rgb(255, 255, 255);
border-width: 2px;
border-color: yellow;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>276</width>
     <height>138</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/plain/img/plain color 2.png);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QCommandLinkButton" name="commandLinkButton">
      <property name="text">
       <string>Register</string>
      </property>
      <property name="icon">
       <iconset resource="Resurse.qrc">
        <normaloff>:/signup/img/signup.png</normaloff>:/signup/img/signup.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_username">
          <property name="text">
           <string>Choose an username</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_password">
          <property name="text">
           <string>Choose a password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_check_password">
          <property name="text">
           <string>Re-enter the password</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="lineEdit_username">
          <property name="styleSheet">
           <string notr="true">border-image: transparent;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_password">
          <property name="styleSheet">
           <string notr="true">border-image: transparent;</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_password_check_password">
          <property name="styleSheet">
           <string notr="true">border-image: transparent;</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>701</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/title/img/title.jpg);</string>
   </property>
  </widget>
  <zorder>graphicsView</zorder>
  <zorder>pushButton</zorder>
  <zorder>groupBox</zorder>
 </widget>
 <resources>
  <include location="Resurse.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>RegisterWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>619</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>620</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
